web/static/js/components/questionnaires/QuestionnaireIndex.jsx

```diff
@@ -141,7 +141,7 @@ class QuestionnaireIndex extends Component<any> {
               if (!questionnaire) return <tr key={-index} className='empty-row'><td colSpan={readOnly ? 3 : 5} /></tr>
 
               return (
-                <tr key={questionnaire.id}>
+                <tr key={questionnaire.id} title={questionnaire.description}>
```

Comment:
the `tr` title is the description but is not showing in the list

<img width="1109" alt="not_showing_description" src="https://user-images.githubusercontent.com/13237343/73662093-fa25f480-4679-11ea-967a-a92141ec3238.png">

Reply:
sorry my bad! I thought it should appear as a subtitle of the questionnaire not with the hover

The description is showing well with the hover 👌 
<img width="1109" alt="hover_description" src="https://user-images.githubusercontent.com/13237343/73662817-39087a00-467b-11ea-8d08-4a4b7e43bceb.png">